spanner/tests/system/test_system.py

```diff
@@ -637,6 +681,26 @@ def _check_sql_results(self, snapshot, sql, params, param_types, expected):
             sql, params=params, param_types=param_types))
         self._check_row_data(rows, expected=expected)
 
+    def test_execute_sql_returning_array_of_struct(self):
+        SQL = (
+            "SELECT ARRAY(SELECT AS STRUCT C1, C2 "
+            "FROM (SELECT 'a' AS C1, 1 AS C2 "
+            "UNION ALL SELECT 'b' AS C1, 2 AS C2) "
+            "ORDER BY C1 ASC)"
```

Comment:
The double quotes delimit the Python strings:  the single quotes delimit the SQL strings.